Packs/HashiCorpTerraform/Integrations/HashiCorpTerraform/HashiCorpTerraform.py

```diff
@@ -0,0 +1,356 @@
+            if not workspace_id:
+                raise DemistoException(
+                    "Please provide either, the instance param 'Default Workspace Id' or the command argument 'workspace_id'"
+                )
+        url_suffix = f'/runs/{run_id}' if run_id else f'/workspaces/{workspace_id}/runs'
+        response = self._http_request('GET', url_suffix, params=params)
+
+        return response
+
+    def run_action(self, run_id: str, action: str, comment: str | None = None) -> Response:
+
+        return self._http_request(
+            'POST',
+            f'runs/{run_id}/actions/{action}',
+            json_data={'comment': comment} if comment and action != 'force-execute' else None,
+            headers=self._headers | {'Content-Type': 'application/vnd.api+json'},
+            ok_codes=[200, 202, 403, 404, 409],
+            resp_type='response')
+
+    def get_plan(self, plan_id: str, json_output: bool) -> Response:
+        url_suffix = f'/plans/{plan_id}{"/json-output" if json_output else ""}'
+        return self._http_request('GET', url_suffix, resp_type='response')
+
+    def list_policies(self, organization_name: str | None = None, policy_kind: str | None = None,
+                      policy_name: str | None = None, policy_id: str | None = None) -> dict:
+
+        params = {}
+        if not policy_id:
+            if policy_kind:
+                params['filter[kind]'] = policy_kind
+            if policy_name:
+                params['search[name]'] = policy_name
+            organization_name = organization_name or self._default_organization_name
+            if not organization_name:
+                raise DemistoException(
+                    "Please provide either the instance param '\
+                        'Default Organization Name' or the command argument 'organization_name'")
+
+        url_suffix = f'/policies/{policy_id}' if policy_id else f'/organizations/{organization_name}/policies'
+        response = self._http_request('GET', url_suffix, params=params)
+
+        return response
+
+    def list_policy_sets(self, organization_name: str | None, policy_set_id: str | None,
+                         versioned: str | None, policy_set_kind: str | None, include: str | None,
+                         policy_set_name: str | None, page_number: str | None,
+                         page_size: str | None) -> dict:
+        params: dict[str, str] = {}
+        if not policy_set_id:
+            if versioned:
+                params['filter[versioned]'] = versioned
+            if policy_set_kind:
+                params['filter[kind]'] = policy_set_kind
+            if include:
+                params['include'] = include
+            if policy_set_name:
+                params['search[name]'] = policy_set_name
+            if page_number:
+                params['page[number]'] = page_number
+            if page_size:
+                params['page[size]'] = page_size
```

Comment:
you can also do it via `assign_params()`.
This message is true for other parts of the code, but this is the best example.

Reply:
`assign_params` is problematic here as there is special characters and saved words at the keys
for example: `params['filter[versioned]'] = versioned` should be `assign_params(filter[versioned]=versioned)`